src/main/kotlin/app/revanced/patches/tiktok/interaction/downloads/patch/DownloadsPatch.kt

```diff
@@ -21,12 +22,14 @@ import app.revanced.patches.tiktok.interaction.downloads.fingerprints.ACLCommonS
 import app.revanced.patches.tiktok.interaction.downloads.fingerprints.ACLCommonShareFingerprint3
 import app.revanced.patches.tiktok.interaction.downloads.fingerprints.DownloadPathParentFingerprint
 import app.revanced.patches.tiktok.misc.settings.fingerprints.SettingsStatusLoadFingerprint
+import app.revanced.patches.tiktok.misc.settings.patch.TikTokSettingsPatch
 import org.jf.dexlib2.Opcode
 import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
 import org.jf.dexlib2.iface.instruction.ReferenceInstruction
 import org.jf.dexlib2.iface.reference.StringReference
 
 @Patch
+@DependsOn([TikTokSettingsPatch::class])
```

Comment:
This patch depends on the integrations because it references the integrations below in the smali code, but does not reference `TikTokSettingsPatch` anywhere.

Reply:
This patch is same as above.

Reply:
This patch does depend on `TikTokIntegrationsPatch`, the same way `TikTokSettingsPatch` does. Both should have the patch in the dependency.